<launch>
  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen">
    <param name="image_width" type="int" value="320" />
    <param name="image_height" type="int" value="240" />
    <param name="framerate" type="int" value="30" />
  

    <param name="autofocus" type="bool" value="False" />
    <param name="focus" type="int" value="0" />
    <!-- other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->

    <param name="video_device" type="string" value="/dev/video1" />
    <!--param name="camera_info_url" type="string" value="file://$(find uvc_camera)/video1.yml" /-->
    <remap from="usb_cam/image_raw" to="usb_camera/image_raw"/>
    <remap from="usb_cam/camera_info" to="usb_camera/camera_info"/>
  </node>

   <!--arg name="ros_namespace" default="usb_camera" /-->

 <!--node ns="$(arg ros_namespace)" pkg="image_proc" type="image_proc" name="image_proc" output="screen"/-->

</launch>
